model/import/README.md

```diff
@@ -3,7 +3,7 @@
 Here you can find instructions about import items using TAO platform.
 
 ## Configuration
-The feature is disabled by default, to enable it flag `FEATURE_FLAG_TABULAR_IMPORT_ENABLED` should be set at the env variables.
+The feature is disabled by default. To enable it, flag `FEATURE_FLAG_TABULAR_IMPORT_ENABLED` should be set to True at the env variables.
```

Comment:
For the Score Setup, lets add a new option:

```
- *none*: In case there is only empty `choice_N_score` fields and no `correct_answer`.
```

And lets fix this one:

```
- *match_correct*: In case there are empty `choice_N_score` and `correct_answer` is provided.
```